pkg/framework/aries/framework.go

```diff
@@ -11,6 +11,7 @@ import (
 	"strings"
 
 	"github.com/hyperledger/aries-framework-go/pkg/didcomm/messenger"
+	"github.com/hyperledger/aries-framework-go/pkg/secretlock"
```

Comment:
Wrong order! Should be `build-in, 3rd party and local`

Reply:
it's not wrong order unless we separate them by empty lines... this is how `go imports` work and linter doesn't fail with this

Reply:
@Baha-sk you should use `goimports -local github.com/hyperledger/aries-framework-go` or manually sort as @soluchok  mentioned. The linter is supposed to be looking for this too.

`goimports --help`
"-local string put imports beginning with this string after 3rd-party packages"

Reply:
that was the IDE adding the import at the wrong place... it's now fixed